packages/lit-dev-content/site/docs/templates/directives.md

```diff
@@ -1015,6 +1024,90 @@ The `cache` directive caches the generated DOM for a given expression and input
 
 Explore `cache` more in the [playground](/playground/#sample=examples/directive-cache).
 
+
+### keyed
+
+Associates a renderable value with a unique key. When the key changes, the previous DOM is removed and disposed before rendering the next value, even if the value - such as a template - is the same.
```

Comment:
For consistency—although I know you don't love it—use the em dash with no spaces. (If there was an easy way to get markdown-it or eleventy to render these as en dashes with narrow spaces, I'd be down for that, I just don't want to try and insert narrow spaces all over the place by hand.)

```suggestion
Associates a renderable value with a unique key. When the key changes, the previous DOM is removed and disposed before rendering the next value, even if the value—such as a template—is the same.
```